libs/src/services/contracts/GovernedContractClient.js

```diff
@@ -0,0 +1,66 @@
+const web3 = require('web3')
+const ethers = require('ethers')
+const ContractClient = require('./ContractClient')
+
+/**
+ * ABI encodes argument types and values together into one encoded string
+ * @param {Array<string>} types
+ * @param {Array<string>} values
+ */
+const abiEncode = (types, values) => {
+  const abi = new ethers.utils.AbiCoder()
```

Comment:
small but do we need to init new abicoder() each time this is called? instead could we init that as singleton on class init and access each time?